Kickstarter-iOS/Views/Controllers/ChangeEmailViewController.swift

```diff
@@ -251,9 +241,17 @@ internal final class ChangeEmailViewController: UIViewController {
   }
 
   private func resetFields(string: String) {
-    _ = [self.passwordTextField, self.newEmailTextField]
-      ||> UITextField.lens.text .~ string
```

Comment:
Why this change?

Reply:
I just got rid of lenses here, and I remember I saw a warning once saying that, to set texts, it's preferred to use the `%~` operator over `.~`.

Reply:
Ah right - so in this case that isn't necessary because the string is not being evaluated for localization. In other places we do this when referring to `Strings.*` to be sure that we're evaluating the closure against any changes to the environment.